cloud/infrastructure/aws/terraform/modules/aws-nomad-servers/instances.tf

```diff
@@ -10,7 +10,13 @@ resource "aws_instance" "servers" {
   subnet_id              = element(var.subnet_ids, count.index)
   vpc_security_group_ids = var.security_group_ids
   iam_instance_profile   = aws_iam_instance_profile.servers.name
-  user_data              = data.template_file.user_data.rendered
+  user_data = templatefile(
```

Comment:
I think this line needs formatting to the equals lines with the above line.

Reply:
it does indeed!